README.md

```diff
@@ -1668,6 +1668,10 @@ This parameter is only used if the [`verify_config`][] parameter's value is 'tru
 
 Specifies whether to validate the configuration file before notifying the Apache service. Valid options: Boolean. Default: `true`.
 
+##### `default_ext`
+
+Specifies what extension to append on end of custom config file. Set to '' to keep original extension. Default: `.conf`.
```

Comment:
I think the phrase "Set to '' to keep original extension" is odd. Under your use case, you could pass '.common' here instead of having it be part of `$name` so I don't think `''` would be something I would recommend.